src/core/streams/actions/morph.js

```diff
@@ -0,0 +1,67 @@
+import { Idiomorph } from "idiomorph/dist/idiomorph.esm"
+import { dispatch } from "../../../util"
+
+export default function morph(streamElement) {
+  const morphStyle = streamElement.hasAttribute("children-only") ? "innerHTML" : "outerHTML"
+  streamElement.targetElements.forEach((element) => {
+    try {
+      Idiomorph.morph(element, streamElement.templateContent, {
+        morphStyle: morphStyle,
+        callbacks: {
+          beforeNodeAdded,
+          beforeNodeMorphed,
+          beforeAttributeUpdated,
+          beforeNodeRemoved,
+          afterNodeMorphed
+        }
+      })
+    } catch (e) {
```

Comment:
Omar could we remove the wrapping `try/catch`, please? I don't think this should raise any exception so we would be silencing bugs we don't know about.

Reply:
@jorgemanrubia removed